src/components/ListItem/CheckBoxListItem/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import DefaultListItem, { Props } from '@/components/ListItem/DefaultListItem'; // eslint-disable-line no-unused-vars
```

Comment:
we're using both items here, do we really need `eslint-disable-line no-unused-vars`?

Reply:
There is a problem in the TS that it doesn't recognize the Props variable.
![Captura de tela de 2020-12-01 17-09-58](https://user-images.githubusercontent.com/12198683/100791545-26c0af00-33f8-11eb-86a5-60d56bd4fcca.png)